lib/opencensus/plug/trace.ex

```diff
@@ -112,13 +112,19 @@ defmodule Opencensus.Plug.Trace do
       :ocp.with_span_ctx(ctx)
     end
 
-    encoded =
-      :ocp.current_span_ctx()
-      |> :oc_span_ctx_header.encode()
-      |> :erlang.iolist_to_binary()
+    case :ocp.current_span_ctx() do
+      :undefined ->
+        conn
 
-    Logger.metadata(tracespan: encoded)
```

Comment:
This should never be there. This is bug. This code should be on line [#85](https://github.com/opencensus-beam/plug/blob/e48ea8420ef58851d107347c4c02be80d7f7cca7/lib/opencensus/plug/trace.ex#L85) where there will always be running span.

Reply:
I did not change this line from the previous version, but I'm happy to remove it.

Reply:
I am not speaking about this particular line, I am speaking about whole code block.